src/api/spec/models/review_spec.rb

```diff
@@ -38,4 +38,85 @@
     subject { Review.unassigned }
     it { is_expected.to match_array([review_unassigned1, review_unassigned2]) }
   end
-end
\ No newline at end of file
+
+  describe '#accepted_at' do
+    let!(:user) { create(:user) }
+    let!(:review) do
+      create(
+        :review,
+        by_user: user.login,
+        state: :accepted,
+        updated_at: Faker::Time.forward(1)
+      )
+    end
+
+    context 'with a review assigned to and state = accepted' do
```

Comment:
And what about a test

``'with a review assigned to and assigned to state != accepted'``
``'with not review assigned to and state != accepted'``